### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_DeployDacpac_DeployDacpacSummaryScreen_ScreenReader: Jaws does not announce the row, column and header value for the table.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6727]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: e8ccc6575cb0cbce189b6e1450ed57c1d306498b
Electron: 4.2.7
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
Jaws Version: 2019.1907.31 ILM

**Additional Details:**
Tools Name: Jaws
MAS References: MAS1.3.1, MAS4.2.3

**Repro Steps:**
1.Launch Azure Data Studio application.
2.Connect to server with windows credentials.
3.Navigate to Extensions from left menu.
4.Search & install Dacpac extension.
5.Navigate to Connections from left menu.
6.Expand the server connected in step 2.
7.Right click an on prem database and select Data-tier application wizard from Context menu.
8.Select first radio button.
9.Navigate till next button & hit enter.
10.Provide all the details for Deploy Dacpac & hit next.
11.Navigate till Next button & hit enter.
12.Start screen reader as Jaws.
13.Navigate to the table using arrow keys.

**Actual:**
On navigating to each cell of the table using arrow key, Jaws does not announce the row , column or the header value.

**Same issue is observed in the following pages:**
1.Extract Dacpac Summary Screen
Follow the exact same steps mentioned above but select the second radio button.
2.Import Bacpac Summary Screen
Follow the exact same steps mentioned above but select the third radio button.

**Expected:**
On navigating to each cell of the table screen reader should announce its respective column, row and header value.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: (https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
The screen reader user will not know exactly which row or column the data belongs to or which header it corresponds to , user will not be able to track which cell he is reading.

**MAS Reference:**
MAS 1.3.1- https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
MAS4.2.3
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={30e4fb40-dcc1-45b4-b714-5e2c28b23ae8}

**Attachment for Reference:**
[#6727A11y_AzureDataStudio(Schema,Dacpac,Backup)_DeployDacpac_DeployDacpacSummaryScreen_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3496588/6727A11y_AzureDataStudio.Schema.Dacpac.Backup._DeployDacpac_DeployDacpacSummaryScreen_ScreenReader.pptx)

**Comment by fsteffi** [comment]
This issue is specially with jaws

**Comment by kisantia** [comment]
Closing because this is a Jaws specific issue and this works with NVDA.
